### DISCUSSION TITLE
How to navigate/select buffer/pane open in foreground ?

### DISCUSSION THREAD

**Discussion by macparnami** [LazyVim/LazyVim#3913] [Q&A] [answered]
Steps to reproduce:
I opened Lazy Pane by 
Leader l (small caps L)
now moved to background buffer of neotree with C-h or C-k
now I cant select Lazy Buffer/pane again to quit it . it obstruct my view 
I need to select it with mouse only then I can quit it .
Is there any shortcut on buffer navigation 
![Capture](https://github.com/LazyVim/LazyVim/assets/16175117/58e2f210-0532-4ff6-8505-816ab34cd99e)

**Comment by folke** [comment, marked as answer]
`ctrl-ww`

**Reply by macparnami** [reply]
Thanks a bunch
